_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,238 @@
+    networks:
+      - opensearch-net
+  opensearch-dashboards:
+    image: opensearchproject/opensearch-dashboards:latest # Make sure the version of opensearch-dashboards matches the version of opensearch installed on other nodes
+    container_name: opensearch-dashboards
+    ports:
+      - 5601:5601 # Map host port 5601 to container port 5601
+    expose:
+      - "5601" # Expose port 5601 for web access to OpenSearch Dashboards
+    environment:
+      OPENSEARCH_HOSTS: '["https://opensearch-node1:9200","https://opensearch-node2:9200"]' # Define the OpenSearch nodes that OpenSearch Dashboards will query
+    networks:
+      - opensearch-net
+
+  jaeger-collector:
+    image: jaegertracing/jaeger-collector:latest
+    ports:
+      - "14269:14269"
+      - "14268:14268"
+      - "14267:14267"
+      - "14250:14250"
+      - "9411:9411"
+    networks:
+      - opensearch-net
+    restart: on-failure
+    environment:
+      - SPAN_STORAGE_TYPE=opensearch
+      - ES_TAGS_AS_FIELDS_ALL=true
+      - ES_USERNAME=admin
+      - ES_PASSWORD=admin
+      - ES_TLS_SKIP_HOST_VERIFY=true
+    command: [
+      "--es.server-urls=https://opensearch-node1:9200",
+      "--es.tls.enabled=true",
+    ]
+    depends_on:
+      - opensearch-node1
+
+  jaeger-agent:
+    image: jaegertracing/jaeger-agent:latest
+    hostname: jaeger-agent
+    command: ["--reporter.grpc.host-port=jaeger-collector:14250"]
+    ports:
+      - "5775:5775/udp"
+      - "6831:6831/udp"
+      - "6832:6832/udp"
+      - "5778:5778"
+    networks:
+      - opensearch-net
+    restart: on-failure
+    environment:
+      - SPAN_STORAGE_TYPE=opensearch
+    depends_on:
+      - jaeger-collector
+
+  hotrod:
+    image: jaegertracing/example-hotrod:latest
+    ports: 
+      - "8080:8080"
+    command: ["all"]
+    environment:
+      - JAEGER_AGENT_HOST=jaeger-agent
+      - JAEGER_AGENT_PORT=6831
+    networks:
+      - opensearch-net
+    depends_on:
+      - jaeger-agent
+
+volumes:
+  opensearch-data1:
+  opensearch-data2:
+
+networks:
+  opensearch-net:
+```
+
+### Step 2: Start the cluster
+
+Run the following command to deploy the Docker compose YAML file.
+
+```
+docker compose up -d
+```
+To stop the cluster, run the following command:
+
+``` 
+docker compose down
+```
+
+## Step 2: View trace data in OpenSearch Dashboards
+
+After you generate Jaeger trace data you can go to OpenSearch Dashboards to view your trace data.
+
+Go to OpenSearch Dashboards Trace Analytics at [Trace Analytics](http://localhost:5601/app/observability-dashboards#/trace_analytics/home).
+
+
+## Use trace analytics in OpenSearch Dashboards
+
+To analyze your Jaeger trace data in the Dashboards, you need to set up Trace Analytics first. To get started, see [Get started with Trace Analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/get-started/).
+
+### Data sources
+
+You can specify either Data Prepper or Jaeger as your data source when you perform trace analytics.
```

Comment:
We use too many personal pronouns, and it's a bit distracting. Suggestion "...as **the** data source..."